CONTRIBUTING.md

```diff
@@ -0,0 +1,79 @@
+# How to contribute #
+
+All help is appreciated. All contributors are unpaid volunteers. If you really want to see something added to Exiv2 - volunteer!
+
```

Comment:
I'd propose to skip this sentences about volunteering and payments. We never know...

Reply:
I took this from the link you recommended me to read:
http://dev.exiv2.org/projects/exiv2/wiki/Contributing_to_Exiv2

I will remove the sentence and let you add whatever you want later.